pkg/client/client.go

```diff
@@ -56,43 +57,73 @@ func newUnixSocketClient(ctx context.Context, config *Config) (*Client, error) {
 	}, nil
 }
 
-func newNetworkClient(ctx context.Context, config *Config) (*Client, error) {
+func newNetworkClient(ctx context.Context, config *Config) (*Client, error) {  //problem somewhere here
```

Comment:
> The original proposal [about introducing gRPC output in Falco](https://github.com/falcosecurity/falco/blob/master/proposals/20190826-grpc-outputs.md) had as a goal to make the gRPC communication over network secured by default.
> 
> Falco honors this point_down https://github.com/falcosecurity/falco/blob/master/falco.yaml#L421-L440 _IIRC, it's mandatory and can't be disabled in Falco_
> 
> So, which is the compelling use case to make this optional? The fact that it can be difficult to configure it does not justify completely skipping it.
> 
> We would need an extensive discussion before changing this behavior eventually. I won't be against this if there're valid reasons.
> 
> cc @falcosecurity/core-maintainers

This would be mostly for testing purposes, i.e., making Falco (and exporter) an easier tool when it comes to prototyping solutions

Some sanity checks are also easier to make when certificates are ignorable -- questions such as 'am I using the right IP? The right port? Can they actually communicate? Is the gRPC server running correctly? ' which can come up in multi-machine environments, environments with weird firewalls, and so on

This comes from a place of trusting that developers will know that going forward without certificates is not a good idea; while also making it easier to test whether or not certificates are the problem

About the comment -- it slipped my mind, I thought I had cleaned all of them up; sorry! 
(I was trying to figure out why using the correct certificates -- and I checked that they were correct -- was returning a bad certificate error, figured out the problem was in that function somewhere, didn't get much further though)